Jenkinsfile

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env groovy
+@Library('digital-jenkins-library@master')
+def digitalCommands = new com.digital.jenkins.DigitalCommands()
+
+podTemplate(
+        label: 'amazon-sqs-java-messaging-lib',
+        containers: [
+                containerTemplate(
+                        name: 'java',
+                        image: 'java:8-jdk',
+                        ttyEnabled: true,
+                        command: 'cat'
+                ),
```

Comment:
we also need to make the nexus admin password available. There is an example here of how to make it available as an environment variable. This is needed to authentication our publish to nexus.

https://github.com/websdev/prometheus-clients/blob/a8c8827955d9cc0eb5c90466259831a735a5c98d/prometheus-spring-boot-client/Jenkinsfile#L11-L13